src/screens/HyperSwitch/Connectors/ConnectorAccountDetailsHelper.res

```diff
@@ -205,7 +321,7 @@ module ConnectorConfigurationFields = {
           selectedConnector
         />
       }}
-      <RenderConnectorInputFields
+      <RenderConnectorLabel
```

Comment:
Does this Component only render the label and no input/related info?

Reply:
yes only label

Reply:
But the code says it render the inputBox also?

Reply:
Label is an inputField that's why it's an input field

Reply:
Now Is this RenderConnectorLabel required, or we can revert to old?